cmd/pmem-dax-check/main.go

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
I believe, the better place for this code would be under `test/` as it's only used for E2E tests.

Reply:
Currently yes, but this might change. It's a rather useful tool that I have used quite a lot lately also outside of the automated tests. I'd prefer to keep it here, but if you disagree then I'll move it.

Reply:
I would prefer to keep in `test/`, It's good to know what @okartau feels.

Reply:
I moved it.